back_end/src/apis/major.py

```diff
@@ -0,0 +1,76 @@
+from flask_cors import CORS
+from flask import Blueprint, request, jsonify
+from flask_login import login_required, login_user, logout_user, current_user
+
+from ..models.user import User
+
+user_api_bp = Blueprint('user_api', __name__)
```

Comment:
This is for user_api. In your api, you should have `major_api_bp = Blueprint('major_api', __name__)`. 
You should replace `@user_api_bp` with `@major_api_bp`. 
Also, `__init__.py` you should do a import: `from .apis.major import major_api_bp as mapi` and register the blueprint `app.register_blueprint(mapi, url_prefix="/api/majors")`